### ISSUE TITLE
模型训练时间

### ISSUE THREAD

**Issue by wanggei** [PaddlePaddle/PaddleNLP#4343]
### 请提出你的问题

问一下文本分类中的基于预训练模型微调的多分类一般训练时间会有多久，训练集有八十多条数据，每条基本上几百字 标签只有5个

训练执行的代码：
python train.py --model_name_or_path ernie-3.0-bash-zh  --data_dir ./data --output_dir checkpoint --device cpu --learning_rate 3e-5 --early_stopping_patience 4 --max_seq_length 128 --per_device_eval_batch_size 4 --per_device_train_batch_size 4 --num_train_epochs 100 --do_train True --do_eval  True --metric_for_best_model accuracy --load_best_model_at_end --evaluation_strategy epoch --save_strategy epoch --save_total_limit 1

**Comment by wawltor** [comment]
> ### 请提出你的问题
> 问一下文本分类中的基于预训练模型微调的多分类一般训练时间会有多久，训练集有八十多条数据，每条基本上几百字 标签只有5个
> 
> 训练执行的代码： python train.py --model_name_or_path ernie-3.0-bash-zh --data_dir ./data --output_dir checkpoint --device cpu --learning_rate 3e-5 --early_stopping_patience 4 --max_seq_length 128 --per_device_eval_batch_size 4 --per_device_train_batch_size 4 --num_train_epochs 100 --do_train True --do_eval True --metric_for_best_model accuracy --load_best_model_at_end --evaluation_strategy epoch --save_strategy epoch --save_total_limit 1

可以观察一下验证集的效果，如果效果已经收敛了，就可以把任务kill掉，具体训练时间还是要任务的难度和效果

**Comment by wanggei** [comment]
@wawltor 验证集是运行evaluate.py把，我感觉我每次训练都很快，不到10分钟就训练完成了，而且训练时下面的进度条每次都百分之5左右都会训练完成，
![943bd705dd96c17426b4b4720f21233](https://user-images.githubusercontent.com/61741330/210687889-756e2238-119e-4937-aec7-415015ab6a15.jpg)

**Comment by wawltor** [comment]
> @wawltor 验证集是运行evaluate.py把，我感觉我每次训练都很快，不到10分钟就训练完成了，而且训练时下面的进度条每次都百分之5左右都会训练完成， ![943bd705dd96c17426b4b4720f21233](https://user-images.githubusercontent.com/61741330/210687889-756e2238-119e-4937-aec7-415015ab6a15.jpg)

你的训练样本比较少，如果训练时间比较短，同时需要观察多次验证集的效果，看看是否稳定下来了；如果稳定下来就可以关闭任务

**Comment by wawltor** [comment]
我们先暂时关闭该issue，如有疑问可以再次打开该issue。
